pkgs/by-name/ac/actual-server/package.nix

```diff
@@ -77,10 +78,14 @@ let
     outputHashMode = "recursive";
     outputHash =
       {
-        x86_64-linux = "sha256-N31aAAkznncKlygyeH5A3TrnOinXVz7CTQ8/G4QX6hY=";
-        aarch64-linux = "sha256-j7BFAKXi+TKIlmHBjbx6rwaKuAo6gnOlv6FV8rnlld0=";
-        aarch64-darwin = "sha256-YpUQYOLJHYxWuE6ToLFkXWEloAau9bLBvdbpNh8jRZQ=";
-        x86_64-darwin = "sha256-AioO82Y6mK0blSQRhhZZtWmduUcYwyVAewcXEVClJUg=";
+        x86_64-linux = "sha256-71OaB2UjqBaPXATx+KW5U+66puh93yRkZFzyqh/YNOA=";
+        # aarch64-linux = "sha256-j7BFAKXi+TKIlmHBjbx6rwaKuAo6gnOlv6FV8rnlld0=";
+        # aarch64-darwin = "sha256-YpUQYOLJHYxWuE6ToLFkXWEloAau9bLBvdbpNh8jRZQ=";
+        # x86_64-darwin = "sha256-AioO82Y6mK0blSQRhhZZtWmduUcYwyVAewcXEVClJUg=";
+        # x86_64-linux = lib.fakeHash;
+        aarch64-linux = lib.fakeHash;
+        aarch64-darwin = lib.fakeHash;
+        x86_64-darwin = lib.fakeHash;
```

Comment:
Try these

```suggestion
        aarch64-linux = "sha256-RoTQAfKtj61ten/+NLrLAxpAohKspyG+4vZhSdPn+14=";
        aarch64-darwin = "sha256-0mC497MDxrFBO/kNiQmVYK6mSHtTBCCTyelxJKwTkD8=";
        x86_64-darwin = "sha256-5l2V0wCpsJk8XwNuzMDn/PI2vdIPxu3FASr3bNtmAMU=";
```

Reply:
I've updated the hashes for 25.3.1 instead